qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -1657,6 +1656,11 @@ void FitPropertyBrowser::showFitResultStatus(const QString &status) {
   if (status != "success") {
     color = "red";
   }
+  if (text == "cannot reach the specified tolerance in F") {
+    text = "Changes in function value are too small";
+  } else if (text == "cannot reach the specified tolerance in X") {
+    text = "Changes in parameter value are too small";
+  }
```

Comment:
Might it be worth moving this message conversion to it's own function `fitOutputStatusToDisplayMessage` or similar?

Reply:
I think this conversion should be done in `LevenbergMarquardtMinimizer.cpp` to make status consistent between the GUI and python

Reply:
ok, I will look at that now

Reply:
done